src/scripts/screensavermanager.js

```diff
@@ -11,7 +11,7 @@ import './screensavermanager.scss';
 function getMinIdleTime() {
     // Returns the minimum amount of idle time required before the screen saver can be displayed
     //time units used Millisecond
-    return 180000;
+    return userSettings.get('screensaverTime') * 1000;
```

Comment:
The suggestion from https://github.com/jellyfin/jellyfin-web/pull/6165#discussion_r1788527870 was not fully applied.
```suggestion
    return userSettings.screensaverTime() * 1000;
```

Reply:
My bad, change made